packages/api-providers/src/abstracts/relayer/webb-relayer.ts

```diff
@@ -223,7 +223,7 @@ export class WebbRelayer {
         url = `${this.endpoint}/api/v1/leaves/evm/${chainId.toString(16)}/${contractAddress}`;
         break;
     }
-    const req = await fetch(url);
+    const req = await fetch(url, { signal: abortSignal });
```

Comment:
Are there any problems if the abortSignal is not provided?

Reply:
No it can be 'undefined' | 'null'